internal/trace/transport.go

```diff
@@ -22,11 +22,14 @@ import (
 	"sync/atomic"
 )
 
+// requestCount records the number of logged request-response pairs and will
+// be used as the unique id for the next pair.
+var requestCount uint64
```

Comment:
nit: do we need to reset the variable's value back to zero, when we are done with the counting? Currently it's not necessary, it's a package level variable and we may need it in the future.